src/resdk/tables/__init__.py

```diff
@@ -0,0 +1,23 @@
+""".. Ignore pydocstyle D400.
+
+============
+ReSDK Tables
+============
+
```

Comment:
Provide a brief description of the `tables` module. For example "Helper classes for aggregating collection data in ta tabular format. "